.pre-commit-config.yaml

```diff
@@ -1,19 +1,3 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This project isn't under ASF

Reply:
Should we need to add the Astronomer header here? How has it been done in the providers' package, @kaxil ?

Reply:
No, there is no IP in this file, so no point in adding license to it

Reply:
We also don’t need to add license to all files since we already have a top-level LICENSE file